consensus/core/src/test_utils.rs

```diff
@@ -0,0 +1,106 @@
+use std::sync::Arc;
+
+use consensus_config::{AuthorityIndex, Committee, Parameters};
+use parking_lot::RwLock;
+use sui_protocol_config::ProtocolConfig;
+
+use crate::{
+    block::{BlockRef, Round, TestBlock, VerifiedBlock},
+    context::Context,
+    dag_state::DagState,
+    metrics::test_metrics,
+};
+
+/// A helper function to create a test context with a committee of given size
```

Comment:
Let's keep test helpers in the most relevant files possible, instead of having a big test_utils down the road which may make dependency management hard. If the existing test helper methods for Context is not enough, and we can move context_for_test() to context.rs. The other two functions can be leave in this file and maybe we can rename this file to test_dag.rs or something appropriate.